shared/schemas/workflows.yml.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
maybe call the directory "json-schemas"? bc there may be other schemas there later (e.g. GraphQL)
@sir-sigurd wdyt?

Reply:
`schemas/json/...` could also work.

Reply:
If you haven't a strong opinion on different pathname, I'm sticking with the current `shared/schema`. It's just a directory name, you can change it anytime

Reply:
ok cool